sdk/ml/azure-ai-ml/azure/ai/ml/operations/_component_operations.py

```diff
@@ -603,10 +603,11 @@ def _try_resolve_code_for_component(component: Component, get_arm_id_and_fill_ba
             # arm id can be passed directly
             pass
         elif isinstance(component.code, Code) or is_registry_id_for_resource(component.code):
+            # TODO: what if component.code is an anonymous Code object?
```

Comment:
@wangchao1230 have we planned to support:
+ anonymous code asset in component yaml
```
code:
  path: ./
```
+ code asset in SDK
```
component.code = ml_client._code.get("test_code", 1)
ml_client.components.create_or_update(component)
```
+ using remote code along with additional includes

First 2 are not allowed for now according to schema CodeFields